### ISSUE TITLE
Bug in README with lualine

### ISSUE THREAD

**Issue by moberst** [gennaro-tedesco/nvim-possession#33]
When I tried the suggested configuration for lualine, it didn't work.   The suggested setup is below
```
lualine.setup({
    sections = {
        lualine_c = {
            {
                require("nvim-possession").status,
                cond = function()
                    return require("nvim-possession").status() ~= nil
                end,
            },
        },
    }
})
```

I think there were two problems:
* First, when a session is not loaded, the use of `.status()` in the condition seemed to throw an error, since the function does not exist.  
* Second, the first `status` returns a function, not a return value, so nothing gets printed even if the first issue is fixed.

Instead, I found that the following worked well.
```
lualine.setup({
    sections = {
        lualine_c = {
            {
              "require('nvim-possession').status()",
               cond = function()
                   return require("nvim-possession").status ~= nil
               end,
            },
        },
    }
})
```

There are two differences
* The call to the status is in quotations, which I believe prevents it from being called unless the condition is true
* The condition function uses `status` instead of `status()`

Posting here in case this is helpful - happy to open a PR to modify the README accordingly

**Comment by gennaro-tedesco** [comment]
Good morning and thank you for bringing this to my attention!

> When I tried the suggested configuration for lualine, it didn't work.

Let us debug step by step, the suggested configuration does work in my case thus we have to reduce it to a minimal `init.lua` to test. First of all

> First, when a session is not loaded, the use of .status() in the condition seemed to throw an error, since the function does not exist.

This is incorrect, even if a session isn't loaded the function still exists, why wouldn't it? As you can see [here](https://github.com/gennaro-tedesco/nvim-possession/blob/253e52fe26f2816acae68e7e0ced9d89adf3f85a/lua/nvim-possession/init.lua#L26) if no session is loaded the function just returns `nil`. Are you sure you aren't doing some sort of lazy loading in your plugins that delays the loading of this one unless a session is created? 

Second:

> There are two differences
> The call to the status is in quotations, which I believe prevents it from being called unless the condition is true

I don't understand this point - if we assume (and we have to validate this assumption) that the `status` function isn't loaded why would the quotation marks prevent it to be called unless true?

**Comment by moberst** [comment]
Thank you for the response!  It may be an issue with lazy-loading on my part (e.g., not calling `.setup()` until later on), I will see if I can reproduce with a minimal `init.lua`.  

I stumbled upon the version that works (for me) via trial-and-error, and perhaps don't fully understand why it is different.  

Nonetheless, to clarify my speculation - I imagined that checking `.status ~= nill` is effectively checking whether or not the function exists, and checking `.status() ~= nil` is calling the function, which itself returns `nil` if no session is loaded.

I'm going to close the issue and re-open if I can come up with a minimal `init.lua` that reproduces the problem.
